components/Article/Article.styled.js

```diff
@@ -0,0 +1,55 @@
+import styled from "styled-components";
+import Link from "next/link";
+import { css } from "styled-components";
+
+const StyledArticle = styled.article`
+  margin: 0 25px 25px 25px;
+
+  position: relative;
+  background-color: white;
```

Comment:
Just a tiny hint: generally it's best practice not  use `named colors` in production apps - may you like to change it in the future.
By the way this is a really interesting  article about [color format from Josh W. Comeau ](https://www.joshwcomeau.com/css/color-formats/) - enjoy reading 🤓

```suggestion
  background-color: #ffffff;
```